tests/math_functions.cpp

```diff
@@ -7,6 +7,8 @@
 
 TEST_CASE( "fast_floor", "[math]" )
 {
+    int n = 1, d = 0;
+    printf( "%d\n", n / d );
```

Comment:
Strictly speaking (according to the c++ standard), this won't always result in a SIGFPE. It's undefined behavior to divide by 0, so some aggressive compiler might just remove this line altogether (I've seen such compilers). Have you considered `std::raise()` instead?

Reply:
Indeed, but thanks for the tip.  Looks like clang is indeed optimising it away, so I'll switch to `std::raise`.